src/test/ui/rust-2018/future-proofing-locals.stderr

```diff
@@ -52,5 +52,51 @@ error: imports cannot refer to local variables
 LL |     use {T as _, x};
    |                  ^
 
+warning: the item `T` is imported redundantly
```

Comment:
In this file it's better to `allow` the lint.
(It if it's merged with `unused_imports`, it will be allowed automatically though.)